tests/tests.js

```diff
@@ -148,7 +148,7 @@
             assert.expect(9);
             image.src = imageUrl;
 
-            $(image).load(function() {
+            $(image).ready(function() {
```

Comment:
Yes, it breaks the tests - that's [how](https://github.com/kottenator/jquery-circle-progress/issues/112).

I'm pretty sure here `$(image).ready` is similar to `$(document).ready` and it is executed not at the same time as the images are `load`-ed. It kinda makes some sense, but I'm still not sure why it fix the problem.